language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -75,6 +88,18 @@ beforeAll(async () => {
   sandboxPort = parseInt(sandboxPortData);
   console.log('Sandbox listening on port ' + sandboxPort.toString());
 
+  const grpcurlUserArgs = [
```

Comment:
maybe add a todo to replace this by a call to the JSON API once you expose that endpoint?